sources/web/datalab/static/appbar.html

```diff
@@ -30,37 +25,40 @@
       </a>
     </div>
     <div class="btn-group">
```

Comment:
you need the `display:none` on the `btn-group` div, otherwise there will be some padding/margin showing for no reason.